src/main/ApplicationImpl.cpp

```diff
@@ -1306,6 +1305,11 @@ ApplicationImpl::syncOwnMetrics()
     }
     mMetrics->NewCounter({"process", "file", "handles"})
         .set_count(fs::getOpenHandleCount());
+
+    // Update simple timer metrics. This both updates the current value of the
+    // "max" metrics to be the max for the current period and starts a new
+    // period.
+    mMetrics->syncSimpleTimerStats();
```

Comment:
I think this needs to be moved to `syncAllMetrics`. Other subsystems call `syncOwnMetrics` ad-hoc so it will mess with our periods.